manager/integration/Dockerfile

```diff
@@ -21,7 +21,10 @@ RUN curl -sO https://storage.googleapis.com/kubernetes-release/release/$KUBECTL_
     wget -q https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_${ARCH}.zip && \
     unzip terraform_${TERRAFORM_VERSION}_linux_${ARCH}.zip && rm terraform_${TERRAFORM_VERSION}_linux_${ARCH}.zip && \
     mv terraform /usr/bin/terraform && \
-    chmod +x /usr/bin/terraform
+    chmod +x /usr/bin/terraform && \
+    curl -s -o flux_install.sh https://fluxcd.io/install.sh && \
```

Comment:
I assume this is always the latest and stable version of flux?

Reply:
Yes, it always gets the latest release version.